src/components/textarea/textarea.css

```diff
@@ -0,0 +1,40 @@
+@import '../../styles/colors-v2.css';
+@import "../../styles/spacing.css";
+@import "../../styles/typography-v2.css";
+
+.textarea {
+  border: 1px solid var(--mds-grey-12);
+  border-radius: 3px;
+  box-sizing: border-box;
+  font: var(--mds-type-content);
+  height: var(--spacing-x-large);
+  margin: 0;
+  padding: 0 var(--spacing-x-large) 0 var(--spacing-medium);
+  width: 100%;
+}
+
+.textarea::placeholder {
+  color: var(--mds-grey-54);
+}
+
+.textarea:read-only {
+  background-color: var(--mds-grey-3);
+}
+
+.invalid-icon {
+  box-sizing: border-box;
+  pointer-events: none;
+  position: absolute;
+  right: 6px;
+  top: 50%;
```

Comment:
For some odd reason, this is not aligning the invalid icon in the center. If you count the pixels under the icon in Input vs Textarea then its 6px vs 4px. Somehow this `top: 50%` is not actually offsetting the icon 50% of the current height of the textarea's container. Instead its off by 2px. 

I'm not sure what is going on but 2px sounds like a border issue.

Reply:
Oh I see the issue. The textarea's container is not correctly assuming the height of its children.

![image](https://user-images.githubusercontent.com/841084/114460175-492cbb00-9b96-11eb-84fd-8226821ba345.png)

There is a gap between the bottom of the textarea and the start of the bottom margin.

Reply:
I think this is also at the root of the first line alignment issue, I'll try to address this.

Reply:
They ended up being separate issues, but yeah, there's something funky with `textarea` sizing vs `input` sizing that seems to be down to browser implementation. Tweaking the padding and offset seems to have solved both issues though.